client/src/app/nav/nav.component.ts

```diff
@@ -14,20 +13,13 @@ export class NavComponent implements OnInit {
   model: any = {};
   currentUser$: Observable<User | null> = of(null);
 
-  constructor(
-    public accoutService: AccoutService,
-    private router: Router,
-    private toastr: ToastrService
-  ) {}
+  constructor(public accoutService: AccoutService, private router: Router) {}
 
   ngOnInit(): void {}
```

Comment:
I think you should not implement OnInit interface, you are not use ngOnInit method
You should remember, code less bug less :)) haha

Reply:
thanks :D i will improve this line